docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -10,7 +10,7 @@ public static class Program
 
         internal static void RunAll()
         {
-            int samples = 0;
+            /*int samples = 0;
```

Comment:
Why is this code commented? 
Shall we either remove it or keep it un-commented?